tests/console/firewalld.pm

```diff
@@ -54,38 +79,45 @@ sub start_stop_firewalld {
     systemctl('start firewalld');
     # wait until iptables -L can print rules, max 10 seconds
```

Comment:
Remove this line

Reply:
The `script_retry()` is still used here to wait but instead of blindly waiting for long time we're waiting for a moment when the firewall is ready.

Reply:
Agree!
Then I would suggest removing `max 10 seconds`, `script_retry` will wait for max 300 seconds.